projects/1.20-fabric/assets/dynamic-surroundings/dsurround/lang/zh_cn.json

```
@@ -0,0 +1,196 @@
+  "dsurround.config.compassAndClockOptions.compassStyle.tooltip": "指南针渲染风格",
+  "dsurround.config.compassAndClockOptions.scale": "渲染缩放",
+  "dsurround.config.compassAndClockOptions.scale.tooltip": "按指定的值缩放指南针渲染",
+  "dsurround.config.enhancedSounds": "增强声音处理",
+  "dsurround.config.enhancedSounds.tooltip": "用于增强声音处理(混响等)的配置选项",
+  "dsurround.config.enhancedSounds.enableEnhancedSounds": "增强声音处理",
+  "dsurround.config.enhancedSounds.enableEnhancedSounds.tooltip": "是否启用增强声音处理(混响，遮挡等)",
+  "dsurround.config.enhancedSounds.enableMonoConversion": "单声道转换",
+  "dsurround.config.enhancedSounds.enableMonoConversion.tooltip": "是否启用立体声到单声道的动态转换。此功能仅在启用“增强声音处理”时可用。",
+  "dsurround.config.enhancedSounds.backgroundThreadWorkers": "后台工作线程",
+  "dsurround.config.enhancedSounds.backgroundThreadWorkers.tooltip": "用于增强声音处理的后台线程数。请不要随意更改此值，它可能会对性能产生影响。",
+  "dsurround.config.enhancedSounds.enableOcclusionProcessing": "遮挡处理",
+  "dsurround.config.enhancedSounds.enableOcclusionProcessing.tooltip": "是否启用声音遮挡处理(消去被方块挡住的声音)。启用后会产生额外的后台工作进程，也可能会导致低配置计算机出现性能问题。",
+  "dsurround.config.enhancedSounds.reverbRays": "混响射线投射数",
+  "dsurround.config.enhancedSounds.reverbRays.tooltip": "计算混响效果时从声源位置投射出的射线数。较高的值会增加CPU的负载并提高保真度。较低的值会降低CPU的负载并降低保真度。当要调整性能时，这是需要注意的重要设置。",
+  "dsurround.config.enhancedSounds.reverbBounces": "混响射线反射数",
+  "dsurround.config.enhancedSounds.reverbBounces.tooltip": "混响射线最多可以反射几次。较高的值会增加CPU的负载并提高保真度。较低的值会降低CPU的负载并降低保真度。",
+  "dsurround.config.enhancedSounds.reverbRayTraceDistance": "混响射线投射距离",
+  "dsurround.config.enhancedSounds.reverbRayTraceDistance.tooltip": "计算混响时，射线最多可用投射出多远的距离，单位为方块。较高的值会增加CPU的负载并提高保真度。较低的值会降低CPU的负载并降低保真度。",
+  "dsurround.config.otherOptions": "其他选项",
+  "dsurround.config.otherOptions.tooltip": "其他“东西”的配置选项",
+  "dsurround.config.otherOptions.playRandomSoundOnStartup": "启动游戏时播放随机音效",
+  "dsurround.config.otherOptions.playRandomSoundOnStartup.tooltip": "当Minecraft加载完成，是否在显示主界面时播放随机音效",
+  "dsurround.text.soundconfig.vanilla": "内置原版音效",
+  "dsurround.text.soundconfig.subtitle": "字幕 ",
+  "dsurround.text.soundconfig.cull": "剔除",
+  "dsurround.text.soundconfig.block": "阻止",
+  "dsurround.text.soundconfig.cull.help": "启用剔除会限制播放短时间内具有相同资源ID的声音数量，从而降低声音引擎以及耳朵的负载。在生物农场附近会很有用。只适用于由Minecraft声音引擎播放的声音。",
+  "dsurround.text.soundconfig.block.help": "设为阻止将阻止具有资源ID的声音播放。阻止也适用于动态环绕的声音。只适用于由Minecraft声音引擎播放的声音。",
+  "dsurround.text.soundconfig.play.help": "播放音量被缩放系数缩放后的声音。不适用于声音类型音量修饰符。",
+  "dsurround.text.soundconfig.volume.help": "在计算声音的输出音量时应用音量缩放系数。增加会使安静的声音变大;降低会使响亮的声音变小。100%是正常音量。",
+  "dsurround.text.config.soundconfiguration": "配置声音",
+  "dsurround.text.config.soundconfiguration.tooltip": "在模组包中配置单个声音",
+  "dsurround.text.keybind.section": "动态环绕",
+  "dsurround.text.keybind.modConfigurationMenu": "模组配置菜单",
+  "dsurround.text.keybind.individualSoundConfig": "独立声音配置",
+  "dsurround.text.keybind.diagnosticHud": "诊断叠加层",
+  "dsurround.text.reloadassets": "[%1$s\u00a7r] \u00a7a重载配置",

```

```suggestion
  "dsurround.text.reloadassets": "【%1$s\u00a7r】 \u00a7a重载配置",
```
标点符号应使用规范的汉语全角标点符号，具体要求请阅读我们的 [翻译规范](https://cfpa.site/TransRules/) 中 **03.标点符号** 这一节。
诸如这类的问题请在您翻译的所有文件中全部查明并修改。

中括号是这样的，一般来说用全角方括号替代

---

改啦